internal/worker/server.go

```diff
@@ -449,16 +449,16 @@ func (s *Server) requestJob(ctx context.Context, arch string, jobTypes []string,
 		return
 	}
 
-	jobType, status, _, err := s.jobStatus(jobId, nil)
+	jobType, channel, status, _, err := s.jobStatus(jobId, nil)
 	if err != nil {
 		logrus.Errorf("error retrieving job status: %v", err)
 	} else {
-		prometheus.DequeueJobMetrics(status.Queued, status.Started, jobType)
+		prometheus.DequeueJobMetrics(status.Queued, status.Started, jobType, channel)
```

Comment:
I think this function in the metrics is accepting the `tenant` as the argument and so the metrics here are showing
`org-<tenant>` in the metrics. So we need to either handle this in the `DequeueJobMetrics` function or pass in `channel[4:]`